qiskit_experiments/library/calibration/fine_amplitude.py

```diff
@@ -32,74 +32,76 @@
+                       └─────┘       └─────┘ ░ └╥┘
+            measure: 1/════════ ... ════════════╩═
+                                                0
+
+        Here, Cal is the name of the gate which will be taken from the name of the schedule.
+        The user can optionally add a square-root of X pulse before the Cal gates are repeated.
+        This square-root of X pulse allows the analysis to differentiate between over rotations
+        and under rotations in the case of pi-pulses. Importantly, the resulting data is analyzed
+        by a fit to a cosine function in which we try and determine the over/under rotation given
+        an intended rotation angle per gate which must also be specified by the user. The steps
+        to run a fine amplitude calibration experiment are therefore
+
+        .. code-block:: python
+
+            qubit = 3
+            amp_cal = FineAmplitude(qubit)
+            amp_cal.set_schedule(
+                schedule=x90p,
+                angle_per_gate=np.pi/2,
+                add_xp_circuit=False,
+                add_sx=False
+            )
+            amp_cal.run(backend)
+
+        Note that the schedule and angle_per_gate could have been set by independently calling
+        :meth:`set_experiment_options` for the schedule and :meth:`set_analysis_options` for
+        the angle_per_gate.
+
+        Error amplifying experiments are most sensitive to angle errors when we measure points along
+        the equator of the Block sphere. This is why users should insert a square-root of X pulse
+        before running calibrations for :math:`\pm\pi` rotations. Furthermore, when running
+        calibrations for :math:`\pm\pi/2` rotations users are advised to use an odd number of
+        repetitions, e.g. [1, 2, 3, 5, 7, ...] to ensure that the ideal points are on the equator
+        of the Bloch sphere. Note the presence of two repetitions which allows us to prepare the
+        excited state. Therefore, add_xp_circuit = True is not needed in this case.
+
+        **Summary of experiment options**
+
+        * repetitions: A list with the number of times the gate of interest will be repeated.
+        * schedule: The schedule of the gate that will be repeated.
+        * add_sx: A boolean which if set to True will add a square-root of X before the repetitions
+          of the gate of interest. Set this to True if you are calibrating gates with an ideal
+          rotation angle per gate of :math:`\pm\pi`. The default value is False.
+        * add_xp_circuit: A boolean which if set to True will cause the experiment to run an
+          additional circuit with an X gate and a measurement. This prepares the excited state
+          and is typically crucial to get the correct sign for the magnitude of the error in
+          the rotation angle. The default value is True.
+        * sx_schedule: Allows users to set a schedule for the square-root of X gate.
+
+        **Summary of analysis options**

```

Comment:
Please move them to `_default_options` of attached analysis class.

Reply:
See here: eac66ce